messaging/src/test/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessorTest.java

```diff
@@ -395,11 +394,12 @@ void exceptionWhileHandlingEventAbortsWorker() throws Exception {
         });
     }
 
+    // todo: callback needed!
```

Comment:
Question: is this todo still valid? It doesn't seem as if there are to many changes in this test.

Reply:
I event don't remember what is it about :D Removed :)